validation/start.go

```diff
@@ -53,6 +53,10 @@ func main() {
 	if err != nil {
 		util.Fatal(err)
 	}
+	outputData, outputErr := ioutil.ReadFile(output)
+	// check the output
+	util.SpecErrorOK(t, outputErr != nil || string(outputData) != "process called\n", specerror.NewError(specerror.ProcArgsApplyUntilStart, fmt.Errorf("`process.args` MUST NOT be applied until triggered by the start operation"), rspecs.Version), outputErr)
```

Comment:
If `outputErr is nil`, it means the process is applied.
In this case, when `string(outputData) != "process called\n"`, we could only get this conclusion:
```
the process is applied, but not running as expected.
```
It is different with ProcArgsApplyUntilStart.
I suggest we can just check the existence of the output file.

Reply:
updated, PTAL.